packages/web/src/scss/settings/_utilities.scss

```
@@ -27,6 +27,15 @@ $utilities: (
                 )
             ),
     ),
+    'background-gradient': (
+        responsive: true,
+        property: background-image,
+        class: bg-gradient,
+        values: (
```

i would prefer to add here a map, but without skeleton a basic-overlay

```suggestion
        values: map.remove(tokens.$gradients, skeleton, basic-overlay)
```

It will allow them to generate any other gradient too.